doc/grahom.xml

```diff
@@ -274,14 +274,35 @@ Transformation( [ 3, 1, 5, 4, 5 ] )
     <Ref Oper="HomomorphismsDigraphsRepresentatives"/>, except they only return
     <E>injective</E> homomorphisms.
     <Example><![CDATA[
-gap> gr1 := ChainDigraph(3);;
-gap> gr2 := Digraph([[3, 5], [2], [3, 1], [], [4]]);
-<immutable digraph with 5 vertices, 6 edges>
-gap> MonomorphismsDigraphs(gr1, gr2);
-[ Transformation( [ 1, 5, 4, 4, 5 ] ), 
-  Transformation( [ 3, 1, 5, 4, 5 ] ) ]
-gap> MonomorphismsDigraphsRepresentatives(gr1, CompleteDigraph(3));
-[ Transformation( [ 2, 1 ] ) ]
```

Comment:
I think you accidentally replaced the example of MonomorphismsDigraphs with examples of SubdigraphsMonomorphisms.

Reply:
Fixed!